examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        view="user_features",
</pre><p>guess <code>view</code> should be <code>user_attributes</code> ?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
@jperez999 and @karlhigley changing view to `user_attributes` give error since it was saved to feature store as `user_features`. so we should leave it as it is.

Reply:
yeah you cant change the name of the view, that comes from the feast configs at the bottom of notebook 1.

Reply:
You can change it if it’s changed in both places though, right?